src/main/webapp/scripts/itinerary-overview.js

```diff
@@ -283,34 +330,34 @@ function isReadyPlaces(){
 function renderPlaceCards(places) {
     let placeCards = [];
     for(let i = 0; i < places.length; i++) {
-        const { phoneNumber, name, address, website } = places[i];
+        const { phoneNumber, name, photoUrl, address, website } = places[i];
 
         placeCards.push(
             `
         <div class="col s12">
             <div class="card horizontal">
-                <div class="card-image">
-        <img src="../assets/img/Building.jpg">
-                </div>
         <div class="card-stacked">
             <div class="card-content">
             ` +
             (name
-                ? `<p>${name}</p>`
+                ? `<h5 style="font-size: large;">${name}</h5>`
                 : '') +
+            (photoUrl
+                ?`<img src="${photoUrl}" style="width: 50px;height: 50px; float: right; border-radius: 8px;" alt="${name}">`
+                : '')+
             (address
-                ? `<p>${address}</p>`
+                ? `<h6 style="font-size: medium;">${address}</h6>`
```

Comment:
Same here, set a class for each of these <h6> tags

Reply:
For some reason, the class doesn't get recognized in the CSS file when I try this so I just went with a working solution